spdockit-spo/office-365-groups-and-onedrive/office-365-management-actions.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Office 365 Management Actions 
+description: This article explains how to use Manage Groups and Membership ribbon to easily manage Office 365 Groups and Teams on the selected object. 
+author: Tomislav Sirovec
+date: 14/11/2017
+---
+
+This article explains how to use Manage Groups and Membership ribbon to easily manage Office 365 Groups and Teams on the selected object.  
+Management buttons are enabled only in __live mode__ of Permissions Explorer and when Groups and Teams node is selected.
+
+### Groups
+
+__Create__ button will create a new Office 365 group. Fields to fill in are: Name, Group ID, Description (not required), Privacy and Owner.
+- Available on: Groups and Teams node of the given tenant.
+- Scope: entire tenant.
+
+__Delete__ button will delete selected group. You can pick a group from the main report while on Groups and Teams node, or you can select it from a list inside the Groups and Teams node.
+- Available on: Groups and Teams node of the given tenant.
+- Scope: group is deleted on entire tenant.
+
+### Membership
+
```

Comment:
Add instructions how can these users be imported - by email, name or using csv or txt file?
For every wizard and action where this is possible.